cron/cron.txt

```diff
@@ -10,4 +10,4 @@
 20      4       1       *       *       bash /home/ubuntu/code/CantusDB/cron/postgres/monthly.sh
 30      4       1       1       *       bash /home/ubuntu/code/CantusDB/cron/postgres/yearly.sh
 40      4       1       *       *       bash /home/ubuntu/code/CantusDB/cron/management/manage.sh populate_next_chant_fields; bash /home/ubuntu/code/CantusDB/cron/management/manage.sh populate_is_last_chant_in_feast
-50      4       *       *       0       /usr/local/bin/docker-compose run --rm certbot certonly --webroot --webroot-path /var/www/certbot/ -d cantusdatabase.org --renew-hook "nginx -s reload"
+50      4       *       *       0       /usr/local/bin/docker-compose run --rm certbot certonly --webroot --webroot-path /var/www/certbot/ -d cantusdatabase.org www.cantusdatabase.org --renew-hook "nginx -s reload"
```

Comment:
I don't know anything about certbot, so I don't know that I can approve. Did it work? When I quickly checked the docs, it looks like they pass multiple domains differently than you do here: 

https://eff-certbot.readthedocs.io/en/stable/using.html#certbot-commands